app/frontend/src/assets/images/images.tsx

```diff
@@ -90,61 +90,73 @@ function svgWrapper(
   };
 }
 
-// COP Icons
+/// COP Icons
 const CopIconData = svgWrapper(CopIconDataP, {
   title: "Data Science Community of Practice Logo",
-  desc: "placeholder",
+  desc: "icon representing the scatter plot and line graph"
 });
 
 const CopIconEngineering = svgWrapper(CopIconEngineeringP, {
   title: "Engineering Community of Practice Logo",
+  desc: "an engineering icon that represents the terminal"
 });
 
 const CopIconOps = svgWrapper(CopIconOpsP, {
   title: "Ops Community of Practice Logo",
+  desc: "an operations icon that represents a running operation being managed"
 });
 
 const CopIconProduct = svgWrapper(CopIconProductP, {
   title: "Product Management Community of Practice Logo",
+  desc: "a product icon that represents a team communicating with each other"
 });
 
 const CopIconUiux = svgWrapper(CopIconUiuxP, {
   title: "UI/UX Community of Practice Logo",
+  desc: "an icon for uiux representing a variety of ideas"
 });
 
 // CTJ Logos
 const ctjLogoTitle = "Civic Tech Jobs Logo";
 
 const LogoHorizontal = svgWrapper(LogoHorizontalP, {
   title: ctjLogoTitle,
+  desc: "civic tech jobs horizontal logo"
 });
 
 const LogoHorizontalOnDark = svgWrapper(LogoHorizontalOnDarkP, {
   title: ctjLogoTitle,
+  desc: "a simple black rectangle - horizontal"
```

Comment:
I'm not sure this is an accurate description. The confusion might stem from the image having white text over a transparent background?

Reply:
hmm. I rerendered the image. Earlier it looked like a black box, but now it looks exactly how you described it. I updated the description. thoughts?